source/mir/combinatorics/package.d

```diff
@@ -251,6 +300,9 @@ Params:
 
 Returns:
     Forward range, which yields the permutations
+
+See_Also:
+    $(LREF Permutations), $(LREF dispose)
```

Comment:
I just tested and if I click on `dispose` it will get to such a page - is this good enough or should we trust that the unittest does the explanation?

![image](https://cloud.githubusercontent.com/assets/4370550/14567575/5b2cef92-033d-11e6-8fba-96ccd58c8996.png)